docs/branch_mgmt.md

```diff
@@ -88,7 +88,7 @@ A scratch view will be opened, showing the diff between the selected branch and
 
 #### Diff history of selected and active branches (`H`)
 
-A scratch view will be opened, showing the commit diff between the selected branch and the active branch.
+A scratch view will be opened, showing the commit diff between the selected branch and the active branch. To cherry-pick a commit, move one courser over the commit you want to cherry-pick and press `c`. The commit might not appear at once then refresh the view  by pressing `r`.
```

Comment:
This is probably okay, but as I have mentioned before, I have a preference for discoverable UX.  Since there is no dashboard to remind users of the key-bindings here, how would you feel about them pressing `Enter` or `Super+Enter`, and then being asked what they want to do with that commit?  It could be:

- view commit
- checkout commit
- cherry-pick commit
- ??? maybe something else